src/shogun2-core/src/main/java/de/terrestris/shogun2/service/PluginService.java

```diff
@@ -95,4 +146,14 @@ public String getPluginSource(String simpleClassName) throws Exception {
 	public void setDao(D dao) {
 		this.dao = dao;
 	}
+
+	/**
+	 * Sets the applicationService.
+	 *
+	 * @param applicationService
+	 */
+	@Autowired
```

Comment:
I tend to **not** like this, but will obey, as you are the expert and we do it elsewhere as well.

When does it make sense to have two services? If I have `ProjectApplicationService` because of something in my project, I should remove the generic bean for `ApplicationService`, right? We as a frame work shoul have as less dependencies on project XML as possible IMO.